pkg/module/drift/drift.go

```diff
@@ -248,30 +248,39 @@ func handleFilters(sel *goqu.SelectDataset, res *ResourceConfig) *goqu.SelectDat
 
 var idRegEx = regexp.MustCompile(`(?ms)^\$\{sql:(.+?)\}$`)
 
-func handleIdentifier(identifiers []string) (exp.Expression, error) {
-	switch l := len(identifiers); {
-	case l == 0:
+const idSeparator = "|"
+
+func handleIdentifiers(identifiers []string) (exp.Expression, error) {
```

Comment:
would document this function and its inner logic, I have some changes I think that would be better to implement such as defining the table and doing table.Col instead of c + "." + name etc', but we can do this in a later PR.